localisation/ko/docs/02-javascript.md

```diff
@@ -0,0 +1,40 @@
+# 02: JavaScript 프론트엔드 개발
+
+## 시나리오
+
+Contoso는 다양한 야외 활동 제품을 판매하는 회사입니다. Contoso의 마케팅 부서는 기존 고객과 잠재 고객에게 제품을 홍보하기 위한 마이크로 소셜 미디어 웹사이트를 런칭하고자 합니다.
+
+JavaScript 개발자로서, Python 백엔드 API 앱과 통신하는 React를 사용하여 JavaScript 프론트엔드 앱을 구축할 것입니다.
+
+## 전제 조건
+
+준비를 위해 [README](../README.md) 문서를 참조하세요.
+
+## 시작하기
+
+- [GitHub Copilot 에이전트 모드 확인](#github-copilot-에이전트-모드-확인)
+- [커스텀 지시사항 준비](#커스텀-지시사항-준비)
+- [애플리케이션 프로젝트 준비](#애플리케이션-프로젝트-준비)
+- [Figma MCP 서버 준비](#figma-mcp-서버-준비)
+- [Figma에서 UI 컴포넌트 생성](#figma에서-ui-컴포넌트-생성)
+- [FastAPI 백엔드 앱 실행](#fastapi-백엔드-앱-실행)
+- [React 프론트엔드 앱 구축](#react-프론트엔드-앱-구축)
+- [React 프론트엔드 앱 확인](#react-프론트엔드-앱-확인)
+
+### GitHub Copilot 에이전트 모드 확인
+
+1. GitHub Codespace 또는 VS Code 상단의 GitHub Copilot 아이콘을 클릭하고 GitHub Copilot 창을 여세요.
+
+   ![Open GitHub Copilot Chat](./images/setup-02.png)
```

Comment:
This image path should point to the original one, which is `../../../docs/images/setup-02.png`.